apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -85,8 +85,13 @@ test.describe("Delegate flows", () => {
           await app.accounts.navigateToAccountByName(account.delegate.account.accountName);
 
           await app.account.clickBannerCTA();
-          await app.delegate.verifyProvider(account.delegate.provider);
-
+          await app.delegate.verifyFirstProviderName(account.delegate.provider);
+          if (account.delegate.account.currency.name === "Solana") {
+            await app.delegate.verifyContinueDisabled();
+            await app.delegate.selectProvider(1);
```

Comment:
wouldn't it be better for readability to select provider by name here? if list of provider is stable then always select Ledger one for example, if list of provider changes then get all, get first one and then select it by name, in this way in report it would be clear what we selected